ci-scripts/setup-cluster.sh

```diff
@@ -358,6 +360,182 @@ $CONDITIONAL_FIELDS
     server_port: 8080
     prometheus_port: 9090
 EOF
+      else
+        oc create secret generic logging-loki-s3 \
+  --from-literal=bucketnames="${AWS_BUCKET_NAME}" \
+  --from-literal=endpoint="https://s3.eu-west-1.amazonaws.com" \
+  --from-literal=region="eu-west-1" \
```

Comment:
We can put both the `endpoint` and `region` values inside some Environment variable for easier change for local testing and in CI:
```
--from-literal=endpoint="https://s3.eu-west-1.amazonaws.com" \
  --from-literal=region="eu-west-1" \
```